cmake/modules/FindALUMINUM.cmake

```diff
@@ -0,0 +1,52 @@
+# Defines the following variables:
+#   - ALUMINUM_FOUND
+#   - ALUMINUM_LIBRARIES
+#   - ALUMINUM_INCLUDE_DIRS
+#
+# Also creates an imported target ALUMINUM
+
+# Find the header
+find_path(ALUMINUM_INCLUDE_DIRS common.h
```

Comment:
I think we should look for "allreduce.hpp" instead. First, "common.h" is a common name; second, it shouldn't even be in the Aluminum project (nor should nccl.h, btw) since it's just the CUDA/NCCL_CHECK macros.

Reply:
Some of Al got refactored; nccl.h is gone.